sdc_services/models/r2/observation.py

```diff
@@ -0,0 +1,34 @@
+"""
+Observations modeled after SDC $extract operation
+
+See http://build.fhir.org/ig/HL7/sdc/extraction.html#observation-based-extraction
+"""
+
+class Observation(object):
+    def __init__(self, derived_from=None, value=None, issued=None):
+        # FHIR reference to other resource
+        self.derived_from = derived_from
+        self.value = value
```

Comment:
Do we not need to maintain a `code`, that is a `CodeableConcept` for what this observation refers to?

Reply:
Done! See below commit *Add Questionnaire code to Observations as Observation.code*

ea2bba55b2a2a40ed8bfa9f700c39aa9dbc3c0a8